relational_operators/BuildHashOperator.hpp

```diff
@@ -148,6 +149,30 @@ class BuildHashWorkOrder : public WorkOrder {
         hash_table_(DCHECK_NOTNULL(hash_table)),
         storage_manager_(DCHECK_NOTNULL(storage_manager)) {}
 
+  /**
+   * @brief Constructor for the distributed version.
+   *
+   * @param input_relation The relation to build hash table on.
+   * @param join_key_attributes The IDs of equijoin attributes in
+   *        input_relation.
+   * @param any_join_key_attributes_nullable If any attribute is nullable.
+   * @param build_block_id The block id.
+   * @param hash_table The JoinHashTable to use.
+   * @param storage_manager The StorageManager to use.
+   **/
+  BuildHashWorkOrder(const CatalogRelationSchema &input_relation,
+                     std::vector<attribute_id> &&join_key_attributes,
```

Comment:
Does the ``WorkOrderFactory`` reconstruct the ``join_key_attributes`` for every new ``BuildHashWorkOrder``?